app/routes/map-feature/zoning-map-index.js

```diff
@@ -0,0 +1,12 @@
+import Route from '@ember/routing/route';
+
+export default class ZoningMapIndexRoute extends Route {
+  async model(params) {
+    const { id } = params;
+
+    return {
+      id,
+      title: id === '30d-Subplan' ? 'SUB PLAN OF 30D' : id,
```

Comment:
The "title" showing up as "SUB PLAN OF 30D" is correct for that sub plan but there are others. We need to detect "subplan" in the value and then do some string manipulation to reformat. That could happen here or in the SQL (@horatiorosa is doing it in [SQL for layers API ](https://github.com/NYCPlanning/labs-layers-api/pull/363/files#diff-35910763243dc92157bf495f807ddf36292ee30c22ae052ad0f334091f35de56R17) but his implementation isn't totally correct as of this writing. Using the literal value of `sectionalm`  for "id", and therefore as the value in the URL, is fine.

Reply:
Done.